tempesta_fw/http.c

```diff
@@ -1150,6 +1305,9 @@ tfw_http_conn_fwd_unsent(TfwSrvConn *srv_conn, struct list_head *eq)
 	list_for_each_entry_safe_from(req, tmp, fwd_queue, fwd_list) {
 		if (!tfw_http_req_fwd_single(srv_conn, srv, req, eq))
 			continue;
+		/* Stop forwarding if the request is in streaming. */
```

Comment:
It seems that next attempt to forward next request will be successful, even if the previous request still be in not finished streamed state - as we unconditionally do srv_conn->msg_sent = (TfwMsg *)req; in tfw_http_req_fwd_single().